portfolio/src/main/webapp/script.js

```diff
@@ -1,30 +1,95 @@
-function scrollUp() {
+
+/** Shows a popup window. */
+function showPopup(button) {
```

Comment:
It looks like the only aspect of the button you're using is the id, so you could probably just pass that id in as a parameter instead.  Then in the HTML code, you can just pass the specific ID for each instead of "this".